### DISCUSSION TITLE
How to verify the script and turning it into an integration?

### DISCUSSION THREAD

**Discussion by abdullahdevrel** [windmill-labs/windmill#1756] [Questions] [answered]
I have submitted the IPinfo API script to the windmill hub.

https://hub.windmill.dev/scripts/ipinfo/1598/get-ip-address-insights-from-ipinfo---geolocation%2C-privacy-detection%2C-company-and-more-ipinfo

Would it be possible to verify the script and turn it into an integration?

**Comment by rubenfiszel** [comment, marked as answer]
Only team members can approve scripts. I reviewed your script, transformed ipinfo token into a resource types and approved it.

**Reply by abdullahdevrel** [reply]
Thanks @rubenfiszel Appreciate it 👍
